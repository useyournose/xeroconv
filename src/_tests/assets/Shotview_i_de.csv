"55grs FOX - 23,30grs D073.4"
#,TEMPO (MPS),Δ DURCHSCHNITT (MPS),KE (J),LEISTUNGSFAKTOR (N⋅S),ZEIT,ÖLSCHUSS,ERSTSCHUSS,SCHUSSNOTIZEN
1, 924.3, 5.7, 1522.4, 3.3, 10:08:59, , , "" 
2, 918.7, 0.1, 1504.2, 3.3, 10:09:14, , , "" 
3, 916.3, -2.3, 1496.2, 3.3, 10:09:29, , , "" 
4, 915.0, -3.6, 1492.1, 3.3, 10:09:49, , , "" 
-,,,,,,
AVERAGE SPEED,918.6,,,,,,,
STANDARDABWEICHUNG,3.5,,,,,,,
STREUUNG,9.2,,,,,,,
AVERAGE POWER FACTOR,3.3,,,,,,,
GEWICHT DES PROJEKTILS(GRAN),55.0,,,,,,,
TRAININGSNOTIZ,"",,,,,,,
-,,,,,,
DATUM, "8. Februar 2025 um 10:08",,,,,
Alle Schüsse sind in den Berechnungen enthalten,,,,,,,﻿