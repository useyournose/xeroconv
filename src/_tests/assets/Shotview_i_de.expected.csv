sep=;
Device ID;useyournose-xeroconv;;

Series No;0001;;
Total number of shots;0004;;

Units velocity;m/s;;
Units distances;m;;
Units kinetic energy;j;;
Units weight;grains (grs);;

Stats - Average;918.575;m/s;
Stats - Highest;924.3;m/s;
Stats - Lowest;915;m/s;
Stats - Ext. Spread;3.3;m/s;
Stats - Std. Dev;9.2;m/s;

Shot ID;V0;Ke0;Proj. Weight;Date;Time
0001;924.3;1522.4;55;08-02-2025; 10:08:59;
0002;918.7;1504.2;55;08-02-2025; 10:09:14;
0003;916.3;1496.2;55;08-02-2025; 10:09:29;
0004;915;1492.1;55;08-02-2025; 10:09:49;
